docker/initialize-fr.sh

```diff
@@ -1,45 +1,33 @@
 #!/bin/bash
 
 PROJECT=$1
+DEP=$2
 
-die() {
-  echo $*
-  exit 1
-}
+if [ -z ${DEP} ]; then
+  BAN="http://bano.openstreetmap.fr/data/full.sjson.gz"
+else
+  BAN="http://bano.openstreetmap.fr/data/bano-${DEP}.json.gz"
+fi
+wget $BAN -O ${GEOCODER_DATA_DIR}/addresses-fr/BAN_odbl.sjson.gz
 
-set -e
-
-[[ -z "${PROJECT}" ]] && die "You must pass a project name in parameter. For example: $0 geocoder"
-
-rm -f ${GEOCODER_DATA_DIR}data-fr/*
-docker service update ${PROJECT}_redis-server-fr
+echo "Creating Addok building service."
 
-wget https://bano.openstreetmap.fr/BAN_odbl/BAN_odbl.json.bz2 -O ${GEOCODER_DATA_DIR}addresses-fr/BAN_odbl.json.bz2
+ADDOK_HOST=redis-server-fr
+ADDOK_ATTRIBUTION=BANO
+ADDOK_LICENCE=ODbL
```

Comment:
Those variables are not necessary as they are only used in service creation. You should directly set it during service creation.

```
--env ADDOK_HOST=redis-server-fr
```

Reply:
more readable as I propose given that the code is similar with the initialize-lu one